### DISCUSSION TITLE
CaptureElement or other UI Element to preview stream from a capture device

### DISCUSSION THREAD

**Discussion by BernardoPoiares** [microsoft/WindowsAppSDK#652] [General] [answered]
Hello,

I want to know if the CaptureElement will be available to use or if it will be replaced for some other new element.

**Comment by andrewleader** [comment, marked as answer]
Hi @BernardoPoiares, great question, however this looks like a question for the WinUI team. You can ask them here: https://github.com/microsoft/microsoft-ui-xaml/issues

Thanks!
